docs/paddle-ocr.md

```
@@ -0,0 +1,141 @@
+
+
+## How to Enable Paddle OCR
+
+### Docker Compose Deployment for Paddle OCR
+
+#### Self-Hosted Deployment Configuration
+
+1. **Prerequisites**: Follow the [deployment guide](deploy-docker-self-hosted.md) up to and including the step labelled "Start all required NIMs."
+
+2. **Configure Environment Variables**:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)